### DISCUSSION TITLE
Policy management

### DISCUSSION THREAD

**Discussion by Mat-Moo** [DependencyTrack/dependency-track#4163] [General]
If you change the rules for a policy, is there a way to ger projects to re-evaluate without upload a sbom again?

**Comment by fupgang** [comment]
Probably not at the moment, as #3951 requests this feature.

**Comment by fupgang** [comment]
From the discussion in #2146:

> Re-evaluation would happen in any of these cases:
> 
>     Upon next SBOM upload
>     Upon manual reanalysis of the project
>     At least 1-2x a day during recurring analysis

Manual reanalysis may refer to this:

![grafik](https://github.com/user-attachments/assets/8bbaf899-3f65-4c37-a645-c5d855c5872a)

**Reply by Mat-Moo** [reply]
Yep, I thought that was doing it, but there is no way to see what is going on - a logs page to show something is being processed would be ever so helpful

**Reply by fupgang** [reply]
Sorry, it actually refers to this on the "audit vulnerabilities" tab
![grafik](https://github.com/user-attachments/assets/cc576987-305d-4ae6-a83c-5102b5b46e49)
The above screenshot triggers a metrics update only.

**Comment by carloreggiani** [comment]
A global reapply policy feature could be useful: in case of a change in the policies, how to complete an assessment on all the project portfolios?
